lua/nvim-tree/populate.lua

```diff
@@ -80,6 +80,21 @@ end
 
 local function gen_ignore_check()
   local ignore_list = {}
+
+  local function add_toignore(path)
+    local content = utils.read_file(path)
+    for s in content:gmatch("[^\r\n]+") do
+      ignore_list[s] = true
+    end
+  end
+
+  if vim.g.nvim_tree_gitgnore > 0 then
```

Comment:
typo `gitIgnore`
also if you don't set the value, we get an error because comparing nil with number :)
`if (vim.g.nvim_tree_gitignore or 0) > 0 then`